node/CHANGELOG.md

```diff
@@ -13,15 +13,28 @@ All notable changes to this project will be documented in this file.  The format
 
 ## Unreleased
 
+### Security
```

Comment:
Could the order of the sections be made to match the commented example lines above please?